pkg/apis/serving/v1alpha2/framework_tensorflow.go

```diff
@@ -15,8 +15,11 @@ package v1alpha2
 
 import (
 	"fmt"
+	"strconv"
 	"strings"
 
+	"k8s.io/klog"
```

Comment:
nit: to be consistent as other places we are using `logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"`

Reply:
done